modules/installation-configuration-parameters.adoc

```diff
@@ -757,6 +757,10 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
 
+|`platform.aws.propagateUserTags`
+| A flag that directs in-cluster operators to include the specified user tags in the tags of the AWS resources that the operators create.
```

Comment:
You could update "operators" to "Operators". This applies to both instances of the word in this line. For more information, see [Operator capitalization](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization) in the OpenShift documentation guidelines.